### ISSUE TITLE
Differentiate focus and focus-keyboard on actions

### ISSUE THREAD

**Issue by skjnldsv** [nextcloud/nextcloud-vue#704]
When navigating with the mouse, the actions are making any .focusable action, well, focused. Meaning it also open, the datepicker.

I think we should have to set of focusable, if the event originate from a keyboard event, then we search for `.focusable-keyboard` else `.focusable-mouse`.
And we keep the `.focusable` for the two.

**Comment by TheCrowned** [comment]
Is the info here still valid? Not sure if I am looking at the right things, but I don't see any `focusable` class in the datepicker:
```
<li class="expirationDateMenu-view20-16">
			<span class="expirationDateContainer-view20-16 icon-expiredate menuitem">
				<label for="expirationDatePicker-view20-16" class="hidden-visually" value=""></label>
				<input id="expirationDatePicker-view20-16" class="datepicker hasDatepicker" type="text" placeholder="Expiration date" value="26-09-2020">
			</span>
		</li>
```

**Comment by juliushaertl** [comment]
@skjnldsv I've been discussing this with @enochterrymize and we are not sure what this is actually referring to. Is it about the date picker in actions that should not open on mouse hover rather just on click focus or through keyboard navigation focus?
